features/toolbar/components/ToolbarContainer.native.js

```diff
@@ -22,25 +23,40 @@ export class ToolbarContainer extends Component {
     render() {
         var underlayColor = ColorPalette.buttonUnderlay;
         var micButtonStyle;
+        var micIconStyle;
```

Comment:
I know it's totally consistent with the codying style of the existing file which I appreciate very much!

I'd like to use the moment though to mention with relation to this PR that we seem to be using `let` rather than `var` elsewhere.